langchain/vectorstores/pinecone.py

```diff
@@ -160,13 +162,27 @@ def similarity_search(
         )
         return [doc for doc, _ in docs_and_scores]
 
-    def _similarity_search_with_relevance_scores(
```

Comment:
@raymond-yuan in the cases where `_similarity_search_with_relevance_scores` is removed (due to incorrect implementation), do we instead want to re-implement to ensure backwards compatibility?

Reply:
We have a default implementation of `_similarity_search_with_relevance_scores` in the base class that will ensure backwards compatibility while also actually computing the relevance score: https://github.com/hwchase17/langchain/blob/117e9efa4fac341671c31bab892111011e034437/langchain/vectorstores/base.py#L189-L213 (not just the distance). 

All classes that previously had this ` _similarity_search_with_relevance_scores` implemented in the subclass should have a default relvance score function now that will be used